src/apps/administration/index.md

```diff
@@ -0,0 +1,73 @@
+<script setup>
+import SwagLanding from "../../components/SwagLanding.vue";
+import SwagLandingCard from "../../components/SwagLandingCard.vue";
+import SwagLandingCardList from "../../components/SwagLandingCardList.vue";
+import * as HeroIcons from "@heroicons/vue/24/solid";
+</script>
+
+<SwagLanding>
+    <template #title>Administration</template>
+    <template #description>
+        <p>Shopware's administration manages all aspects of your store. It outlines all internal processes like product inventory tracking, order processing, customer record management, etc. The Administration component is tightly coupled with Shopware's core, which helps to manage administrative activities.</p>
+        <h1>Capabilities</h1>
+    </template>
+    <template #im>
```

Comment:
Please, rename the slot name from `im` to `image`. We can also use `image` property on the `SwagLanding` component.